mix.exs

```diff
@@ -19,6 +20,11 @@ defmodule ExDoc.Mixfile do
     [clean: [&clean_test_fixtures/1, "clean"]]
   end
 
+  defp package do
+   [licenses: ["Apache 2.0"],
+    links: %{"GitHub" => "https://github.com/elixir-lang/ex_doc"}]
```

Comment:
@milmazz hahaha, sorry I swear I wasn't pulling your leg with my last comment, for some reason I always thought we were talking about this line:
https://github.com/elixir-lang/ex_doc/pull/367#discussion-diff-39346741R23
which to me seemed like a fixed
but now i see they were two different fields.